KTOR-6782 Add an example of the PlaceHolderList class usage (#557)
ktorio/ktor-documentation: topics/server-html-dsl.md

3. A child template will look as follows:
   ```kotlin
   ```
   {src="snippets/html-templates/src/main/kotlin/com/example/Application.kt" include-lines="47-62"}

   This template exposes the `articleTitle`, `articleText` and `list` properties, whose values will be inserted inside the `article`.

> 4. To provide list values as the template we need another class: 

Reviewer: ```suggestion
4. To provide a list of values as the template, create the following new class: 
```

   ```kotlin
   ```
   {src="snippets/html-templates/src/main/kotlin/com/example/Application.kt" include-lines="64-83"}

>    This template uses `PlaceholderList` class to build unordered list (`UL`) from provided items.

Reviewer: ```suggestion
   This template uses the `PlaceholderList` class to generate an unordered list (`UL`) from the provided items.
```

>    It surrounds first item in the list with `<b>` element.

Reviewer: ```suggestion
   It also wraps the first item in a `<b>` element for emphasis.
```
